File: app/src/main/java/com/johnpetitto/orachat/ui/chats/ChatsView.java
package com.johnpetitto.orachat.ui.chats;

import java.util.List;

public interface ChatsView {
    void showLoading(boolean show);
    void showError();
    void displayChatsByDate(List<Object> chatsGroupedByDate);
    void showMoreChatsByDate(List<Object> chatsGroupedByDate);
    void pageLoaded();
}

File: app/src/main/java/com/johnpetitto/orachat/ui/chats/ChatsView.kt
package com.johnpetitto.orachat.ui.chats

interface ChatsView {
    fun showLoading(show: Boolean)
    fun showError()
    fun displayChatData(chatData: List<Any>)
    fun showMoreChatData(chatData: List<Any>)
    fun pageLoaded()
}
